### PR TITLE
Account for multi module maven project

### PR DESCRIPTION
**Pull request by jenniferboedker**
addresses #324

### CODE REVIEW

**Comment by jenniferboedker** [comment]
@KochTobi can the PR be merged? Or do you request further changes?

**Comment by KochTobi** [comment]
I cannot get it to run locally. Maybe someone else is able to do that.

**Review by Steffengreiner — DISMISSED** [review]
LGTM, i was able to get it to run locally after the changes to the entrypoint in the portlet.xml 🚀 
A question would be if this should also be done for the osgi-groovy portlet creation

**Comment by Steffengreiner** [comment]
To clarify, the qube creation works, but the classpath of `life.qbic.portal.portlet.QBiCPortletUI` had to be changed since it was already defined in the portal-utils-lib [dependency](https://github.com/qbicsoftware/portal-utils-lib/blob/master/src/main/java/life/qbic/portal/portlet/QBiCPortletUI.java) causing IntelliJ to indicate a duplicate class name. 
Bildschirmfoto 2021-06-11 um 13 22 08

File: qube/create/templates/portlet/portlet_groovy/{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}-app/src/main/groovy/life/qbic/portal/portlet/PortletUI.groovy [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+@SuppressWarnings("serial")
+public abstract class PortletUI extends UI {
+
+    /**
+     * Location of the properties file where maven stores version/repository url.
+     */
+    public static final String PORTLET_PROPERTIES_FILE_PATH = "portlet.properties";
+
+    /**
+     * Location of the local configuration. This is useful when testing portlets loocally using {@code mvn jetty:run}.
```
Comment (Steffengreiner) [inline qube/create/templates/portlet/portlet_groovy/{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}-app/src/main/groovy/life/qbic/portal/portlet/PortletUI.groovy]:
```suggestion
     * Location of the local configuration. This is useful when testing portlets locally using {@code mvn jetty:run}.
```

File: qube/create/templates/portlet/portlet_groovy/{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}-app/pom.xml [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>{{ cookiecutter.project_slug }}</artifactId>
+        <groupId>life.qbic</groupId>
+        <version>{{ cookiecutter.version }}</version> <!-- <<QUBE_FORCE_BUMP>> -->
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>war</packaging>
+    <artifactId>{{ cookiecutter.project_slug }}-app</artifactId>
+    <dependencies>
```
Comment (Steffengreiner) [inline qube/create/templates/portlet/portlet_groovy/{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}-app/pom.xml]: Bildschirmfoto 2021-06-11 um 16 31 10
I just noticed that the url is referring to the app module and not the parent which breaks the linkage: 
One could think about adding the parent module url here akin to the commit to [sample-tracking](https://github.com/qbicsoftware/sample-tracking-status-overview/pull/3/commits/f5053401c1271ffb4ff03b860a58587036e0a379):

**Review by Steffengreiner — DISMISSED** [review]
LGTM! I found a minor typo and a breakage in the url footer but this can be addressed in a follow-up PR 🚀

File: qube/create/templates/portlet/portlet_groovy/{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}-app/pom.xml (hunk 2 above)
Comment (jenniferboedker) [reply qube/create/templates/portlet/portlet_groovy/{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}-app/pom.xml]: I commited your suggestion :)

**Review by Steffengreiner — APPROVED** [review]
Very nice! 🚀

**Pull request merged**